docs/apache-airflow-providers-docker/decorators/docker.rst

```diff
@@ -0,0 +1,130 @@
+api_version
+    Remote API version. Set to ``auto`` to automatically detect the server's version.
+container_name
+    Name of the container. Optional (templated)
+cpus:
+    Number of CPUs to assign to the container. This value gets multiplied with 1024.
+docker_url
+    URL of the host running the docker daemon.
+    Default is unix://var/run/docker.sock
+environment
+    Environment variables to set in the container. (templated)
+private_environment
+    Private environment variables to set in the container.
+    These are not templated, and hidden from the website.
+force_pull
+    Pull the docker image on every run. Default is False.
+mem_limit
+    Maximum amount of memory the container can use.
+    Either a float value, which represents the limit in bytes,
+    or a string like ``128m`` or ``1g``.
+host_tmp_dir
+    Specify the location of the temporary directory on the host which will
+    be mapped to tmp_dir. If not provided defaults to using the standard system temp directory.
+network_mode
+    Network mode for the container.
+
+    It can be one of the following:
+        bridge
+            Create new network stack for the container with default docker bridge network
+        'None'
+            No networking for this container
+        container:<name> or <id>
+            Use the network stack of another container specified via <name> or <id>
+        host
+            Use the host network stack. Incompatible with `port_bindings`
+        '<network-name>' or '<network-id>'
+            Connects the container to user created network(using `docker network create` command)
+tls_ca_cert
+    Path to a PEM-encoded certificate authority to secure the docker connection.
+tls_client_cert
+    Path to the PEM-encoded certificate used to authenticate docker client.
+tls_client_key
+    Path to the PEM-encoded key used to authenticate docker client.
+tls_hostname
+    Hostname to match against the docker server certificate or False to disable the check.
+tls_ssl_version
+    Version of SSL to use when communicating with docker daemon.
+tmp_dir
+    Mount point inside the container to
+    a temporary directory created on the host by the operator.
+    The path is also made available via the environment variable
+    ``AIRFLOW_TMP_DIR`` inside the container.
+user
+    Default user inside the docker container.
+mounts
+    List of mounts to mount into the container, e.g.
+    ``['/host/path:/container/path', '/host/path2:/container/path2:ro']``.
+working_dir
+    Working directory to set on the container (equivalent to the -w switch the docker client)
+xcom_all
+    Push all the stdout or just the last line. The default is False (last line).
+docker_conn_id
+    ID of the Airflow connection to use
+dns
+    Docker custom DNS servers
+dns_search
+    Docker custom DNS search domain
+auto_remove
+    Auto-removal of the container on daemon side when the container's process exits.
+    The default is False.
```

Comment:
IMHO, [The default](https://github.com/apache/airflow/blob/b52ce7f483d02fe17467ca3fa13cfffdb92ac103/airflow/providers/docker/operators/docker.py#L134) seems `never`.

Reply:
I will check this and update. @tiokim

Reply:
I verified and the default value is False @tiokim